bolt/include/bolt/Passes/IdenticalCodeFolding.h

```diff
@@ -36,6 +36,14 @@ class IdenticalCodeFolding : public BinaryFunctionPass {
 
   const char *getName() const override { return "identical-code-folding"; }
   Error runOnFunctions(BinaryContext &BC) override;
+
+private:
+  /// Create a skip list of functions that should not be folded.
+  Error createFoldSkipList(BinaryContext &BC);
```

Comment:
I think a skip list is a well known data structure in literature, so it's confusing.